BuildScript.Nuke/MSBuildProperties.cs

```diff
@@ -34,4 +34,8 @@ public static class MSBuildProperties
   public const string TargetFrameworkVersion = "TargetFrameworkVersion";
   public const string TargetFrameworks = "TargetFrameworks";
   public const string AssemblyInfoFile = "AssemblyInfoFile";
+  public const string DocumentationFileName = "DocumentationFileName";
+  public const string DocumentationRootPage = "DocumentationRootPage";
+  public const string DocumentationNamespaceSummaryFiles = "DocumentationNamespaceSummaryFiles";
```

Comment:
I don't think that those are well known MSBuild properties, but rather properties used by sandcastle. Those should be moved to the sandcastle constants.